tests/components/StatusBar/StatusBar.spec.tsx

```diff
@@ -0,0 +1,30 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+import * as React from "react";
+import {mount} from "enzyme";
+import {StatusBar} from "../../../src/components/StatusBar";
+import {initStore} from '../../../src/actions/AppActions';
+import appStore from '../../../src/stores/AppStore';
+
+initStore();
+const project = appStore.getProject().getModel();
+const wrapper = mount(<StatusBar/>);
```

Comment:
can you move this out of module global to function inside "tests for statusbar"?  it is expected that the mount will be called only when an "it" body is called.